catalyst/runners/__init__.py

```diff
@@ -3,4 +3,11 @@
 from catalyst.runners.runner import Runner
 from catalyst.runners.supervised import SupervisedRunner
 
-__all__ = ["Runner", "SupervisedRunner"]
+from catalyst.settings import IS_HYDRA_AVAILABLE
+
+if IS_HYDRA_AVAILABLE:
+    from catalyst.runners.hydra_supervised import HydraSupervisedRunner
+
+    __all__ = ["Runner", "SupervisedRunner", "HydraSupervisedRunner"]
+else:
+    __all__ = ["Runner", "SupervisedRunner"]
```

Comment:
In the case of renaming we also could simply this to:
```
__all__ = ["Runner", "SupervisedRunner", "MultiSupervisedRunner"]
```

Reply:
Ok. It is possible.